registry/api/errcode/errors.go

```diff
@@ -23,8 +22,13 @@ func (ec ErrorCode) ErrorCode() ErrorCode {
 	return ec
 }
 
-// Error returns the ID/Value
+// Error returns the human readable version of the ErrorCode
 func (ec ErrorCode) Error() string {
+	return ec.Message()
+}
+
+// ID returns the ID/Value
+func (ec ErrorCode) ID() string {
```

Comment:
So, the string value of the error code is the Descriptor.Value field, which is typically something like `UNKNOWN`.